### ISSUE TITLE
lodash set?

### ISSUE THREAD

**Issue by tony-kerz** [javadev/underscore-java#223]
hi,

thanks for bringing underscore/lodash goodness to java!

any reason that [lodash set](https://lodash.com/docs/4.17.15#set) is missing here?

is there some obvious java native alternative that i'm missing?

best,
tony.

**Comment by javadev** [comment]
Dear Tony,

I like you idea. Lodash set is nice.

Sincerely, Valentyn

**Comment by tony-kerz** [comment]
hi valentyn,

yes, i think it would be a nice addition, i put together this proof-of-concept that might be helpful as a starting point, to be fully compatible it would need to be augmented with array indexing (e.g. `path` of something like `a.b[4].c`):

> it is written in groovy, but you get the point

```
def _set(object, path, value){
  assert path
  if (object != null) {
    def _path = path.tokenize('.')
    if (_path.size > 1) {
      if (!object[_path[0]]) {
        object[_path[0]] = [:]
      }
      _set(object[_path[0]], _path[1..-1].join('.'), value)
    } else {
      object[_path[0]] = value
    }
  }
  object
}
```

regards,
tony.

**Comment by javadev** [comment]
It will be implemented with version 1.50.

**Comment by javadev** [comment]
New version was released today. 🎉
